app/src/main/java/de/othr/plantico/MainActivity.kt

```diff
@@ -6,21 +6,26 @@ import androidx.activity.viewModels
 import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
 import de.othr.plantico.database.PlantApplication
+import de.othr.plantico.databinding.ActivityMainBinding
 
 
 class MainActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityMainBinding
     private val testViewModel: TestViewModel by viewModels {
         TestViewModelFactory((application as PlantApplication).repository)
     }
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        val view = binding.root
+        setContentView(view)
 
         val recyclerView = findViewById<RecyclerView>(R.id.recyclerview)
```

Comment:
This can be removed, right?

Reply:
I mean line 25, val recyclerView = findViewById<RecyclerView>(R.id.recyclerview)

Reply:
Oh yes. Didn`t see that.